### PR TITLE
Update peer-finder docs.

### PR DESCRIPTION
**Pull request by erictune**
More detail on options.
Rename PetSet to StatefulSet

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/2082)

### CODE REVIEW

**Comment by erictune** [comment]
@bprashanth @kow3ns

File: pets/peer-finder/README.md [hunk 1]
```diff
@@ -1,15 +1,39 @@
...
+   send an administrative command to the main container over the localhost network. (Note that signalling
+   is not practical since pods currently do not share a PID namespace).
+3. As pid 1 of the main container, in which case both `--on-change`  and `--on-start` may be used.
+   In this mode, the ordering of the peer-finder relative to the main app is ensured by having the peer
+   finder start the main app.  An example script would be to modify a configuration file and send SIGHUP
+   to the main process.
+4. Both 1 and 2.
+
+Options 1 and 2 and 4 may be preferable since they do not require changes to the main container image.
+Option 3 is useful is signalling is necessary.
```
Comment (bprashanth) [inline pets/peer-finder/README.md]: is useful if signalling

File: pets/peer-finder/README.md [hunk 2]
```diff
@@ -1,15 +1,39 @@
...
+of the PetSet.  It periodically looks up the SRV record of the DNS entry that corresponds to a Kubernetes
+Service which enumerates the set of peers for this  the specified service.
 
+Be sure to use the `service.alpha.kubernetes.io/tolerate-unready-endpoints` on the governing service
+of the StatefulSet so that all peers are listed in endpoints before any peers are started.
 
+There are several ways to bundle it with your main application.
 
+1. In an [init container](http://kubernetes.io/docs/user-guide/pods/init-container/),
+   to help your pod determine its peers when it it first started (determine the desired set of
```
Comment (bprashanth) [inline pets/peer-finder/README.md]: when it is first started

**Review by bprashanth — COMMENTED** [review]
LGTM sans nits

**Comment by kow3ns** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[pets/peer-finder/README.md, line 6 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/contrib/2082#-KXhF1HIn7e8WqP48Gl4:-KXhEwZZ4gM5cXbpZIm9:b-xtjzx1) ([raw file](https://github.com/kubernetes/contrib/blob/1bf0f27651c5d9b1d50ed310f11a9fb0b4d594b3/pets/peer-finder/README.md#L6)):*
> ```Markdown
> 
> All it does is watch DNS for changes in the set of endpoints that are part of the governing service
> of the PetSet.  It periodically looks up the SRV record of the DNS entry that corresponds to a Kubernetes
> ```

PetSet needs to be replaced.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/contrib/2082)*

**Comment by kow3ns** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/contrib/2082)*

**Comment by foxish** [comment]
Merging. @kow3ns to fix remaining comments.

**Pull request merged**